ops/model.py

```diff
@@ -1243,6 +1303,12 @@ class ModelError(Exception):
     pass
 
 
+class UnknownServiceError(Exception):
```

Comment:
Is this used anywhere?

Reply:
This is used in the new exception handling code above.